plugins/mcreator-localization/help/fr_FR/rangeditem/bullet_model.md

```diff
@@ -1,3 +1,3 @@
-Bullet model is the model of your ranged item.
+Le modèle de balle est le modèle de votre arme à distance en balle.
 
-This parameter will override item for texture parameter when used.
\ No newline at end of file
+Ce paramètre remplacera le paramètre d'item comme texture lorsqu'il est utilisé.
```

Comment:
Should be "Ce paramètre remplacera l'item par le paramètre de la texture quand utilisé."